integr8ly/installation
# Intly 3034 update integreatly installer to reference the node js app

## Additional Information
[JIRA-INTLY-3034](https://issues.jboss.org/browse/INTLY-3034)
The following PR is to verify that integreatly installer instead of refercing the old CRUD app it now references the Node.js app.


## Verification Steps
1. Clone the installation repo https://github.com/R-Lawton/installation.git
2. Checkout the Intly 3034 update integreatly installer to reference the node js app branch
3. Delete the old template using the following command `oc delete template spring-boot-rest-http-crud -n openshift`
4. Run the following command to run the role asscoiated with the files modified `ansible-playbook -i inventories/hosts playbooks/install_infra.yml` This allows you to test the new changes without have to uninstall and reinstalll _"Installation"_
5. Once that has ran and completed log into your cluster and navigate to the Solution explorer.
6. From there run through Walkthrough1.
7.The app created at the end should look like the following:

![new_app](https://user-images.githubusercontent.com/53817495/64269503-e131df80-cf31-11e9-95ab-f6ada2be9f61.png)

rather then the old
![original_app](https://user-images.githubusercontent.com/53817495/64269612-1a6a4f80-cf32-11e9-9288-36acb48223be.png)
